blockchains/cryptoorg/assets/0xf78a326ACd53651F8dF5D8b137295e434B7c8ba5/info.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Matic Token",
     "symbol": "MATIC",
-    "type": "CRC20",
+    "type": "CRYPTOORG",
```

Comment:
@cryptomanz1 @vcoolish please review should it be `CRC20` or `CRYPTOORG`

Reply:
CRC20

Reply:
@vcoolish We already have CRC20 for Cronos. Which type should be Cronos tokens?

Reply:
not quite understand. For this kind of cryptoorg should be CRC20
for cryptoorg which is cosmos sdk based should be CRO I suppose, but we don't support it in apps.